modules/ding_dashboard/modules/ding_dashboard_notification/lib/class.phpmailer.php

```diff
@@ -0,0 +1,2397 @@
+<?php
```

Comment:
Third party libraries should not be part of the module.

Why is PHPMailer needed? An if it is it should be loaded using the libraries module.